perfscale_regression_ci/utils/ocp_utils.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#set -ex
+
+cluster_health_check() {
```

Comment:
let's pick either this function or the one in the ocp_utils.py but not leave both since they are duplications of each other